canonical/store-components
# Create CardList component

## Done
Added a `CardList` component

## How to QA
- Go to https://store-components-3.demos.haus/?path=/story/cardlist--default-story
- Check that there is a list of cards (pagination will not work in this PR)

## Issue
Fixes https://warthogs.atlassian.net/browse/WD-1369
